### PR TITLE
Feat/login

### PR DESCRIPTION
**Pull request by vahea95**

### CODE REVIEW

File: src/Components/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      // const user = make request to get profile info
+      if (user) {
+        // User is signed in, see docs for a list of available properties
+        // https://firebase.google.com/docs/reference/js/firebase.User
+        const uid = user.uid;
+        // ...
+      } else {
+        // then user hasn't created his profile redirect to create profile page or make request to create profile with existing data
+        // ...
+      }
```
Comment (ArthurO2002) [inline src/Components/Profile.tsx]: Remove logs and comments

File: src/Components/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    const email = user.email;
+    const photoURL = user.photoURL;
+    const emailVerified = user.emailVerified;
+
+    // The user's ID, unique to the Firebase project. Do NOT use
+    // this value to authenticate with your backend server, if
+    // you have one. Use User.getToken() instead.
+    const uid = user.uid;
+    console.log(user);
+  }
```
Comment (ArthurO2002) [inline src/Components/Profile.tsx]: Remove comments

File: src/Components/Profile.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+export const Profile = () => {
+  const name = "";
+  const surName = "";
+  const email = "";
+  const avatar = "";
+
+  const auth = getAuth();
+  const user = auth.currentUser;
+  if (user !== null) {
```
Comment (ArthurO2002) [inline src/Components/Profile.tsx]: Dellete this component it's not making any sense

File: public/index.html [hunk 4]
```diff
@@ -39,5 +39,8 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script
+            type="text/typescript"
+            ></script>
   </body>
```
Comment (ArthurO2002) [inline public/index.html]: Remove

File: .gitignore [hunk 5]
```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.idea
```
Comment (ArthurO2002) [inline .gitignore]:
```suggestion

// Intelij
.idea
```

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/Components/Profile.tsx (hunk 1 above)
Comment (vahea95) [reply src/Components/Profile.tsx]: done

File: .gitignore (hunk 5 above)
Comment (vahea95) [reply .gitignore]: done

File: public/index.html (hunk 4 above)
Comment (vahea95) [reply public/index.html]: done

File: src/Components/Profile.tsx (hunk 3 above)
Comment (vahea95) [reply src/Components/Profile.tsx]: done

File: src/Components/Profile.tsx (hunk 2 above)
Comment (vahea95) [reply src/Components/Profile.tsx]: done

**Review by ArthurO2002 — APPROVED** [review]

File: .env [hunk 6]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://192.168.1.133:5000/api
```
Comment (ArthurO2002) [inline .env]: ENV should not be pushed

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: .env (hunk 6 above)
Comment (vahea95) [reply .env]: done

**Review by ArthurO2002 — APPROVED** [review]

**Pull request merged**